meta-leda-bsp/conf/machine/qemux86-64-extra.conf

```diff
@@ -10,3 +10,7 @@ MACHINE_ESSENTIAL_EXTRA_RDEPENDS += "v86d"
 MACHINE_EXTRA_RRECOMMENDS = "kernel-module-snd-ens1370 kernel-module-snd-rawmidi"
 WKS_FILE ?= "qemux86-grub-efi.wks"
 do_image_wic[depends] += "syslinux:do_populate_sysroot syslinux-native:do_populate_sysroot mtools-native:do_populate_sysroot dosfstools-native:do_populate_sysroot"
+
+MACHINE_FEATURES:append = " pcbios efi"
```

Comment:
Do we have put them inside yaml file of the distro (leda-kirkstone.yaml)?

Reply:
These machine specific configuration previously were in the local.conf of our distro build repository.
They are supposed to be moved to the machine specific configuration of the meta-leda distro layer.